frontend/src/components/Editor.js

```diff
@@ -5,6 +5,17 @@ import "ace-builds/src-noconflict/mode-csharp";
 import "ace-builds/src-noconflict/theme-monokai";
 import { useFileUpload } from "./fileUploadContext";
 
+function searchFileName(fileId, files) {
+  const result = Object.keys(files).filter((fileName) => {
```

Comment:
this should probably be find instead of filter

Reply:
I actually agree. Both work but find makes a bit more sense

Reply:
Sorry, I'll fix this